backend/src/scripts/send-feedback-form.ts

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs'
+import path from 'path'
+import promptSync from 'prompt-sync'
+import csv from 'csvtojson'
+import axios from 'axios'
+
+const endpoint = '/v1/transactional/govsg/send'
+
+void (async function main() {
+  const prompt = promptSync()
+  const fileName = prompt('Enter .csv file name in temp folder: ')
+  const recipients = (
+    await csv({ output: 'csv' }).fromFile(
+      path.resolve(__dirname, '../../src/temp', fileName)
+    )
+  ).map((r) => r[0]) as string[]
+  const env = prompt('Enter environment (dev/stg/prod): ')
+  if (env !== 'dev' && env !== 'stg' && env !== 'prod') {
+    console.log('Invalid environment')
+    process.exit(1)
+  }
+  const baseUrl =
+    env === 'dev'
+      ? 'http://localhost:4000'
+      : env === 'stg'
+      ? 'https://api-staging.postman.gov.sg'
+      : 'https://api.postman.gov.sg'
+  const apiKey = prompt('Enter API key: ')
+  const axiosInstance = axios.create({
+    baseURL: baseUrl,
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${apiKey}`,
+    },
+  })
+  const newFile = path.resolve(
+    __dirname,
+    '../../src/temp',
+    `${fileName}-response.csv`
+  )
+  for (const recipient of recipients) {
```

Comment:
for reasons I don't understand, I can't get this to work using `.map` or `.forEach`

Reply:
ohh this is why
> Using map or forEach would not allow for the use of await within the loop body, which would result in the loop moving on to the next iteration before the response from the API is received.

Reply:
`map` can, u need to map them to async function executions then do a `await Promise.all`